src/pages/createOrder.js

```diff
@@ -521,6 +528,20 @@ class CreateOrderPage extends React.Component {
     };
   }
 
+  getSEPAMandate() {
+    return this.props.intl.formatMessage(
```

Comment:
We usually use [defineMessages](https://github.com/yahoo/react-intl/wiki/API#definemessages) so [babel-plugin-react-intl](https://github.com/yahoo/babel-plugin-react-intl) knows how to extract the strings to language files.

However I'm not super familiar with that part so maybe it works without it. Can you do a `npm run build:langs` to check if the string is correctly exported to `src/lang/en.json`?